Cargo.toml

```diff
@@ -24,7 +24,9 @@ block-padding = "0.3.3"
 bitflags = "2.3.1"
 
 # extension
-trussed-auth = "0.2.2"
+trussed-auth = "0.3.0"
+trussed-staging = { version = "0.3.0", features = ["chunked", "hkdf"] }
+trussed-chunked = { version = "0.1.0", optional = true }
```

Comment:
Why staging and chunked?  I don’t see it being used in the code.

Reply:
It's needed for the usbip runner because it includes fido. I made sure only what is necessary is now used. Chunked can be removed by disabling large blobs.